rollup.config.js

```diff
@@ -65,16 +66,32 @@ const external = (id) =>
   !id.startsWith('\0') && !id.startsWith('.') && !id.startsWith('/');
 
 // Pragmatically create a Rollup config for each package
-const getRollupConfig = ({ pwd, babelConfigFile }) => {
+const getRollupConfig = ({ pwd, babelConfigFile, isModule }) => {
   const SOURCE_DIR = path.resolve(pwd);
+  const PRIVATE_PATH = `${SOURCE_DIR}/src/private.js`;
+
   // Get the package.json file
   const pkgConfig = require(`${SOURCE_DIR}/package.json`);
+
   // Relative input location for Rollup to bundle from
-  const input = [`${SOURCE_DIR}/src/index.js`, `${SOURCE_DIR}/src/private.js`];
+  const input = [`${SOURCE_DIR}/src/index.js`];
+
+  // Not all packages have private exports
+  if (fs.existsSync(PRIVATE_PATH)) {
+    input.push(PRIVATE_PATH);
+  }
 
-  // Shared Rollup plugins
+  // Shared Rollup plugins for code-only packages
   const rollupPlugins = [nodeResolve(), postcss(getPostCSSOptions()), json()];
 
+  // Gather info about the package
+  const IS_CJS = Boolean(pkgConfig?.main);
+  const IS_ESM = Boolean(pkgConfig?.module);
```

Comment:
```suggestion
  const IS_ESM = pkgConfig?.type === 'module' || Boolean(pkgConfig?.module);
```
Technically we should also be checking the type first to determine if it is ESM. If there is a type then, the package should definitely be a module. Additionally, if `type === 'module'` then we should not have any CJS at all. So that would be the best way to check if ESM_ONLY. Any package that is only a module using type should not have CJS.